<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gestion_joueur</class>
 <widget class="QDialog" name="gestion_joueur">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>661</width>
    <height>459</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="Modifier_joueur">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Modifier un joueur</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>661</width>
     <height>461</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Supprimer_joueur">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Supprimer un joueur</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Afficher_joueur">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Statistiques</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Ajouter_joueur">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="text">
    <string>Ajouter un joueur</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Afficher_joueur_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>30</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Précédent</string>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>Modifier_joueur</zorder>
  <zorder>Supprimer_joueur</zorder>
  <zorder>Afficher_joueur</zorder>
  <zorder>Ajouter_joueur</zorder>
  <zorder>Afficher_joueur_2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
